next-tavla/app/(admin)/edit/[id]/components/TileCard/TransportModeAndLines.tsx

```diff
@@ -0,0 +1,103 @@
+'use client'
+import { TTransportMode } from 'types/graphql-schema'
+import { TTile } from 'types/tile'
+import { TransportIcon } from 'components/TransportIcon'
+import { transportModeNames } from './utils'
+import { Checkbox } from '@entur/form'
+import { TLineFragment } from './types'
+import { useState } from 'react'
+
+function TransportModeAndLines({
+    tile,
+    transportMode,
+    lines,
+}: {
+    tile: TTile
+    transportMode: TTransportMode | null
+    lines: TLineFragment[]
+}) {
+    const lineElements = document.getElementsByName(
+        `${tile.uuid}-${transportMode}`,
+    )
+
+    const anyLineInWhitelist = lines.some((l) =>
+        tile.whitelistedLines?.includes(l.id),
+    )
+    const missingLinesInWhitelist = lines.some(
+        (l) => !tile.whitelistedLines?.includes(l.id),
+    )
+
+    const defaultChecked = () => {
+        if (missingLinesInWhitelist && anyLineInWhitelist)
+            return 'indeterminate'
+        return (
+            !tile.whitelistedLines ||
+            tile.whitelistedLines.length === 0 ||
+            !missingLinesInWhitelist
```

Comment:
hvis tilen ikke har linjer, eller den har lengde 0 eller ikke mangler lines, så vil denne være `true`. det betyr at om man har huket av for å ikke vise noen linjer, så vil de automatisk bli satt til at alle er huket av. stemmer det, og vil det være ønsket oppførsel?

burde ikke denne kun bli `true` om man ikke har noen missing lines, og resterende tilfeller blir `false`?

Reply:
Ja, se kommentar over